tf2_ros/src/tf2_ros/buffer.py

```diff
@@ -1,9 +1,9 @@
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
-# 
+#
```

Comment:
Any unrelated whitespace changes should probably also be revert to reduce the difference to upstream as much as possible.

Reply:
Atom is removing the whitespaces, solved